src/main.cpp

```diff
@@ -8813,7 +8813,7 @@ bool LoadAdminMessages(bool bFullTableScan, std::string& out_errors)
 {
     int nMaxDepth = nBestHeight;
     int nMinDepth = fTestNet ? 1 : 164618;
-    nMinDepth = pindexBest->nHeight - (BLOCKS_PER_DAY*30*6);
+    nMinDepth = pindexBest->nHeight - (BLOCKS_PER_DAY*30*12);
```

Comment:
Reason for this?

Reply:
To avoid situations where a beacon is younger than 6 months but older than 180000 blocks, thus still valid since `GetListOfWithConsensus` uses time, not height.